### DISCUSSION TITLE
Adding progress bar in html-keyboard-response plugin

### DISCUSSION THREAD

**Discussion by batshell** [jspsych/jsPsych#1753] [Q&A]
I have a study where you need to press (say left and right arrow key) and I need to display the representation of the keypresses as a progress bar for both the keys, a rough picture is attached
![sad](https://user-images.githubusercontent.com/59352904/115885381-8aae4780-a46d-11eb-9041-f217ab4bf8f7.png)

**Comment by jodeleeuw** [comment]
Depending on the specifics of how you want this to behave, you might find it easier to do this with the [reconstruction plugin](https://www.jspsych.org/plugins/jspsych-reconstruction/).

**Reply by batshell** [reply]
Can you tell me in more detail on how to put the (two) bars on the extreme left and extreme right while the text content in the middle? because jspsych-conent style puts everything in the center of the screen. Also, I don't want a button, I want need to record a keypress.

**Reply by batshell** [reply]
I had to edit few pre-existing plugins to accommodate two bootstrap progressbar on the page, the code looks very dirty, I will release the code in future (because it's someone's master's project). I think we may be able to add a library for sense of agency and intention based experiments in jspsych.
